### PR TITLE
Allow doctrine-extensions 2.0, updated datasource-bundle dependency

### PR DESCRIPTION
**Pull request by szymach**
Waits for [#304](https://github.com/fsi-open/admin-bundle/pull/304)

### CODE REVIEW

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

File: composer.json [hunk 1]
```diff
@@ -18,9 +18,9 @@
         "symfony/translation": "^2.6|^3.0",
         "symfony/validator": "^2.5|^3.0",
         "fsi/admin-bundle": "^3.0@dev",
-        "fsi/doctrine-extensions-bundle": "^1.1.6",
-        "fsi/doctrine-extensions": "^1.1.6",
-        "fsi/datasource-bundle": "^1.3@dev",
+        "fsi/doctrine-extensions-bundle": "^1.1.6|^2.0@dev",
+        "fsi/doctrine-extensions": "^1.1.6|^2.0@dev",
+        "fsi/datasource-bundle": "^2.0@dev",
```
Comment (chives) [inline composer.json]: why it can't be `^1.3@dev|^2.0@dev` ?

Comment (szymach) [inline composer.json]: because `1.3@dev` does not exist.

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

**Pull request merged**